doc/source/serve/architecture.md

```diff
@@ -24,62 +24,89 @@ There are three kinds of actors that are created to make up a Serve instance:
   the control plane. The Controller is responsible for creating, updating, and
   destroying other actors. Serve API calls like creating or getting a deployment
   make remote calls to the Controller.
-- Router: There is one router per node. Each router is a [Uvicorn](https://www.uvicorn.org/) HTTP
+- HTTP Proxy: There is one HTTP proxy actor on the head node (the location and number of proxies is configurable via the `location` field of `http_options`: {ref}`core-apis`). This actor runs a [Uvicorn](https://www.uvicorn.org/) HTTP
```

Comment:
Specify that _by default_ there's one on the head node. For scalability and HA, it can be run on every node.